src/server/io.rs

```diff
@@ -67,35 +80,29 @@ impl MessageReader for StdioMsgReader {
         }
 
         if res[0].to_lowercase() != "content-length:" {
-            debug!("Header is missing 'content-length'");
-            return None;
+            continue;
```

Comment:
We still only support `Content-Length` here - would you mind also adding a TODO mentioning missing support for the `Content-Type` header?

Reply:
Good point. I will add this support in followup PR then.